.github/workflows/file_formatting.yml

```
@@ -16,4 +16,4 @@ jobs:
         uses: actions/checkout@v4
       - name: verify example_test.go
         run: |
-          grep "^func " example_test.go | sort -c
+          grep "^func [a-z-A-Z]" example_test.go | sort -c
```

This is erroring in the actions run:
`grep: Invalid range end`
Should be
```suggestion
          grep "^func [a-zA-Z]" example_test.go | sort -c
```

---

Would you also update that in the Makefile?